src/abstractdataframe/abstractdataframe.jl

```diff
@@ -863,9 +863,15 @@ end
 
 """
     filter(function, df::AbstractDataFrame)
+    filter(cols => function, df::AbstractDataFrame)
 
 Return a copy of data frame `df` containing only rows for which `function`
-returns `true`. The function is passed a `DataFrameRow` as its only argument.
+returns `true`. The function is passed a `DataFrameRow` as its only argument
```

Comment:
I'd turn this into three sentences (no cols, single col, multiple cols), each starting with ``If `cols` ...``

Reply:
fixed